src/qibo_client/qibo_client.py

```diff
@@ -68,7 +68,8 @@ def run_circuit(
         self,
         circuit: qibo.Circuit,
         device: str,
-        nshots: int = None,
+        nshots: T.Optional[int] = None,
+        verbatim: T.Optional[bool] = False,
```

Comment:
```suggestion
        verbatim: bool = False,
```
Optional means that the default value is `None`